libmscore/measurenumber.cpp

```diff
@@ -57,6 +57,9 @@ QVariant MeasureNumber::propertyDefault(Pid id) const
 
 void MeasureNumber::layout()
       {
+      if (!measure())
```

Comment:
I think I need a comment (not `// Fix #301847.`)  why it might not be `measure`, that this is a normal situation in this case, and not a programming error.

Reply:
The comment is in the commit but I agree, having it in the code is better since it isn't that obvious.